src/admin.c

```diff
@@ -197,11 +201,18 @@ main(int argc, char** argv)
    int option_index = 0;
    size_t command_count = sizeof(command_table) / sizeof(struct pgagroal_command);
    struct pgagroal_parsed_command parsed = {.cmd = NULL, .args = {0}};
+   char* configuration_path = NULL;
+   struct main_configuration* config = NULL;
+   int ret;
+   size_t size;
+   bool remote_connection = false;
+   char* logfile = NULL;
 
    while (1)
    {
       static struct option long_options[] =
       {
+         {"config", required_argument, 0, 'c'},
```

Comment:
isn't `m` less confusing than `c`?

Reply:
`c` is for main configuration file, otherwise it is another letter

Reply:
I have used main configuration file to add the master_key_file_location.

Reply:
I think for `admin.c` we just need the explicit `-m` option e.g. no `-c`